applications/StructuralMechanicsApplication/structural_mechanics_application_variables.cpp

```diff
@@ -68,6 +68,10 @@ KRATOS_CREATE_VARIABLE(double, I33)
 KRATOS_CREATE_VARIABLE(double, LUMPED_MASS_ROTATION_COEFFICIENT)
 KRATOS_CREATE_VARIABLE(Vector, BEAM_INITIAL_STRAIN_VECTOR)
 
+// semi rigid beam variables
+KRATOS_CREATE_VARIABLE(double, ROTATIONAL_STIFFNESS_AXIS_2)
+KRATOS_CREATE_VARIABLE(double, ROTATIONAL_STIFFNESS_AXIS_3)
```

Comment:
just to confirm, those are actual stiffnesses and not bools to enable/disable the hinge?
And they are nodal quantities, not elemental?

Reply:
yes thats correct, its the rotational nodal stiffness in the local direction of the beam